meson_options.txt

```diff
@@ -197,3 +197,10 @@ option(
     value : 'false',
     description : 'Enable live-reloading workflow for stages (for development only)'
 )
+
+option(
+    'integrity_files',
+    type : 'boolean',
+    value : false,
+    description : 'Generate .sig and .sha256sum files of built archives for releases'
+)
```

Comment:
To be honest I don't quite like polluting the list of options with this, if it can be avoided. Maybe we should just call `scripts/sign-release.py` from CI? That would simplify things a bit on the build system side.

Reply:
All done, it's manually run through the script now.